Releasing AC 2.3.4-5

ephraimbuddy opened the pull request.

ashb reviewed (commented):
Missing https://github.com/apache/airflow/pull/26801 in the changelog aren't you?

ephraimbuddy commented:
> Missing [apache/airflow#26801](https://github.com/apache/airflow/pull/26801) in the changelog aren't you?

Yeah true. Added it

> Missing [apache/airflow#26801](https://github.com/apache/airflow/pull/26801) in the changelog aren't you?

Yeah. Thanks. I have added it
